starfish/imagestack/imagestack.py

```diff
@@ -1193,3 +1196,12 @@ def __call__(self, *args, **kwargs):
         for keyword_argument_removed in self.keyword_arguments_removed:
             kwargs.remove(keyword_argument_removed)
         return self.func(*called_args, **kwargs)
+
+
+def apply_finalizer_callable(
```

Comment:
This is only ever used in our code to write to an array, right? I suspect we can be more specific in our naming to make clearer what's going on here. 

```suggestion
def write_result_to_shmem_array(
```

Reply:
This is no longer necessary, given the changes I made to #754